developer/src/kmc/src/commands/buildTestData/index.ts

```diff
@@ -17,7 +18,8 @@ export async function buildTestData(infile: string, _options: any, commander: an
     saveDebug: false,
     shouldAddCompilerVersion: false,
     readerOptions: {
-      importsPath: fileURLToPath(new URL(...LDMLKeyboardXMLSourceFileReader.defaultImportsURL))
+      importsPath: fileURLToPath(new URL(...LDMLKeyboardXMLSourceFileReader.defaultImportsURL)),
+      localImportsPaths: [ dirname(infile) + sep ], // local dir
```

Comment:
This will go wrong for root folders (two separators)

Reply:
Ok good point - is there a better way to indicate that this is a directory? The problem I ran into is that for path `c:\some\dir\file.xml`  dirname was `c:\some\dir` so it resolved to `c:\some\other.xml` instead of `c:\some\dir\other.xml` 

Perhaps an append if it doesn't already end with `sep`? And perhaps a utility function..

Reply:
Use with path.join?

Reply:
Versus path.resolve in the callback - hmm .

That would mean that abs path and ../ would be disallowed - probably ok.

Reply:
helper function may do the job. I'm not sure of a better way. `C:\`, `/` are both problematic, but so is `\\machine\share` which is kinda another root path on Windows (`\\machine` is not really a root because you can't read or write to it)